app/models/container_project.rb

```diff
@@ -30,5 +31,15 @@ def services_count
     container_services.size
   end
 
+  def containers_count
+    container_count = 0
+    container_groups.each do |container_group|
+      container_group.container_definitions.each do |_def|
+        container_count += 1
+      end
+    end
+    container_count
```

Comment:
I have a feeling that there's a more elegant way to do this. @matthewd @tenderlove